### DISCUSSION TITLE
2023년 04월 17일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#36] [General]
# 2023년 04월 17일

LOG 월급을 받아서 토리, 망고, 포이랑 저녁에 표현 식당에 가서 고기를 먹었다. 표현 식당 처음 가봤는데 사장님이 우리 명찰을 보시고 예전에 회식 왔던 사람들이랑 같은 회사(?)냐고 물어보셔서 신기했다 ㅋㅋㅋ 2단계 구현을 완료했다. 구현할 게 …

https://cyma-s.github.io/2023-04-17/

**Comment by Choi-JJunho** [comment]
병원 안갈거면 약 삼시세끼 꼬박꼬박 드세요~
건강이 우선입니다 👍

**Reply by Cyma-s** [reply]
밥도 아니고 약 삼시세끼는 처음이네요~
노력해보겠습니다~

**Comment by greeng00se** [comment]
좋은 명언... 하나 남기고 갑니다... 껄껄껄...

활력을 유지하십시오. 건강이없는 삶은 물이없는 강과 같습니다. 
- 막심 라 가세

**Reply by Cyma-s** [reply]
막심 라 가세의 명언 좋네요~
저도 명언 하나 남기고 갑니다

허브티는 건강에 좋다 - 허브
